cmd/mb.go

```diff
@@ -27,5 +29,9 @@ bucket name should follow DNS name rule with:
 }
 
 func mbRun(_ *cobra.Command, args []string) (err error) {
+	// check zone flag (required)
+	if contexts.Zone == "" {
```

Comment:
maybe we could check this in `ParseFlagIntoContexts`

We can get command name via `cmd.Name()`